.github/workflows/sync-from-sunbeam.yaml

```diff
@@ -0,0 +1,49 @@
+# This workflow is to sync dashboards and alert rules from the Sunbeam repo and create a PR if there is an update.
+name: sync-from-sunbeam
+on:
+  workflow_dispatch:
+  workflow_call:
+  schedule:
+    - cron: '43 1 * * 1'
+
+
+env:
+  REPO_OWNER: canonical
+  REPO_PATH: openstack-exporter-operator
+
+jobs:
+
+  sync-from-sunbeam:
+    name: Pull files from sunbeam
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout local repository
+        uses: actions/checkout@v4
+
+      - name: Pull dashboard and alert rules from sunbeam
+        run: |
+          ./scripts/sync-from-sunbeam.sh
+
+      - uses: actions/create-github-app-token@v1
+        id: app_token
+        with:
+          app-id: ${{ secrets.APP_ID }}
+          owner: ${{ env.REPO_OWNER }}
+          private-key: ${{ secrets.APP_PEM }}
+          repositories: ${{ env.REPO_PATH }}
+      - name: Create pull request
+        uses: canonical/create-pull-request@main
+        with:
```

Comment:
I guess you need to specify: `base: main`. See https://github.com/canonical/create-pull-request?tab=readme-ov-file#inputsbase, and also the pull request created by the workflow (it's targeting the wrong branch)

Reply:
I think we can avoid base: main, because this seems to do weird things when running from a PR branch for testing.

Reply:
that's correct: we probably don't need `base` in create-pull-request action, but will need `base` in checkout action